src/command/remark/NextForecastByCommand.ts

```diff
@@ -0,0 +1,55 @@
+import { format, _ } from "commons/i18n";
+import { UnexpectedParseError } from "commons/errors";
+import { IBaseRemark, RemarkType, Remark } from "../remark";
+import { Command } from "./Command";
+
+export interface INextForecastByRemark extends IBaseRemark {
+  type: RemarkType.NextForecastBy;
+
+  day: number;
+  hour: number;
+  minute: number;
+}
+
+export interface INextForecastByRemarkDated extends INextForecastByRemark {
```

Comment:
It might be too much work, but it would be cool if this type could automatically be provided for type guarding on `RemarkType.NextForecastBy` if using dated taf or forecast parsing. Kind of like `TAFTrendDated`, maybe create and override with a `DatedRemark` or something like that?

Reply:
I've implement that now. Take a look if that works for you.